docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -46,6 +46,7 @@ $ ./bin/yb-tserver --help
 - [Logging](#logging-options)
 - [Raft](#raft-options)
   - [Write Ahead Log (WAL)](#write-ahead-log-wal-options)
+- [Cluster](#cluster-options)
```

Comment:
The other options are also cluster options. I'd prefer to rename to Sharding options.

Reply:
I was wondering about the "Clustering" category since the "Raft" options are also clustering options. These buckets were originally agreed on by Karthik, but as we work with the categories, they have been changing (as expected).